HomeWork_One/Quiz_Template/src/mockData/data.ts

```diff
@@ -0,0 +1,65 @@
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
is the package mandatory?
the thing is that all packages impact the bundle size, and this particular package has a 168 kB size
meaning that users that use your app will download the app + `uuid` to build ids in a runtime
Maybe this is a bit overengeneered thing, isn't it? 🤔